src/tests/ov_helpers/ov_lpt_models/src/concat.cpp

```diff
@@ -758,7 +758,7 @@ std::shared_ptr<ov::Model> ConcatFunction::getOriginalWithIntermediateWithConsta
         attributes.antialias = false;
         attributes.pads_begin = { 0 };
         attributes.pads_end = { 0 };
-        const auto outputShape = op::Constant::create(
+        const auto outputShape = ov::opset1::Constant::create(
```

Comment:
why not concrete operation version? `v0`

Reply:
I decided to leave opset approach for `ov_*_models` because it's simpler for use - no need to determine which version need to use. Anyway, it requires huge additional work, we may do it later

Reply:
For all the tests except GNA I used versions instead of opsets. For GNA I applied the fastest change because tests will be removed soon